content/docs/command-reference/params/diff.md

```diff
@@ -19,21 +19,22 @@ positional arguments:
 
 ## Description
 
-This command provides a quick way to compare parameter values among experiments
-in the repository history. Requires that Git is being used to version the
-project params.
+Provides a quick way to compare parameter values among experiments in the
+repository history. Requires that Git is being used to version the project
+params.
 
-> Parameter dependencies are defined with the `-p` option in `dvc run`. See also
-> `dvc params`.
+> Parameters are defined in the `params` field of `dvc.yaml` (e.g. with the the
```

Comment:
so, here is another example - I think `Parameter dependencies` was the right form

Reply:
I was trying to simplify the language but we do use "param dep" a lot already so I reverted this to not mess with that for now.

Another example of? Sorry, didn't get that part.